pkg/conf/conf.go

```diff
@@ -156,6 +156,24 @@ type Main struct {
 	// Histogram parameters for the processing duration
 	ProcessingDurationBucketFactor float64
 	ProcessingDurationBuckets      int
+
+	// RateLimiter config
+	// Limits the number of records in the time window a container can send to maximum of
+	// RateLimiterContainerRecordLimit records.
+	RateLimiterContainerRecordLimit int
+	// Limits the number of records in the time window that generally can be sent to maximum of
+	// RateLimiterGlobalRecordLimit records.
+	RateLimiterGlobalRecordLimit int
```

Comment:
I think it's worth mentioning in a comment that this is the switch for the global rate limiter.

Reply:
Right. Added.